File: WhichPharmaPortal/WhichPharma.Models/Client/CreateRFQResult.cs
namespace WhichPharmaPortal.Models.Client
{
    public enum CreateRFQResult
    {
        Created,
        ErrorSendingEmails,
        ErrorAccessingDatabase,
        EmailsSentButErrorIntegratingWithStreak,
        EmailsSentButErrorSavingInDatabase,
        RfqNumberIsAlreadyInUse,
        ExistingRfqNotFound,
    }
}

File: WhichPharmaPortal/WhichPharmaPortal/ClientApp/src/models/CreateRFQResult.ts
export enum CreateRFQResult {
	Created = 'Created',
	ErrorSendingEmails = 'ErrorSendingEmails',
	ErrorAccessingDatabase = 'ErrorAccessingDatabase',
	EmailsSentButErrorIntegratingWithStreak = 'EmailsSentButErrorIntegratingWithStreak',
	EmailsSentButErrorSavingInDatabase = 'EmailsSentButErrorSavingInDatabase',
	RfqNumberIsAlreadyInUse = 'RfqNumberIsAlreadyInUse',
	ExistingRfqNotFound = 'ExistingRfqNotFound',
}